app/Models/categories.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Categories extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'name',
```

Comment:
Aquí, es lo que les decía ¿Dónde está descripción?
```suggestion
        'name',
        'description',
```